build.gradle

```diff
@@ -31,7 +31,7 @@ repositories {
 
 dependencies {
   // Re-export classes and interfaces that will be used upstream
-  api 'cloud.eppo:sdk-common-jvm:3.3.1'
+  api 'cloud.eppo:sdk-common-jvm:3.5.0-SNAPSHOT'
```

Comment:
you probably don't need this to be a snapshot version anymore since v3.5.0 has been released, right? same below line 43

Reply:
👍